### PR TITLE
Refactored the views and tests to use the current system date for current tax year.

### PR DESCRIPTION
**Pull request by DavidWoodward37**

### CODE REVIEW

File: app/common/Dates.scala [hunk 1]
```diff
@@ -37,5 +45,22 @@ object Dates {
...
 
   def getYear(date: LocalDate): Int = date.getYear
+
+  def taxYearToString(input: Int): String = {
+    val startYear = input - 1
+    val endYear = input.toString.takeRight(2)
+    s"$startYear/$endYear"
+  }
+
+  def getCurrentTaxYear: Future[String] = {
```
Comment (jameshforster) [inline app/common/Dates.scala]: Should be using LocalDate not DateTime

File: app/common/Dates.scala [hunk 2]
```diff
@@ -20,13 +20,21 @@ import java.time._
...
+
 object Dates {
 
+  val taxYearEnd = "04-05"
   val formatter = DateTimeFormatter.ofPattern("d/M/uuuu").withResolverStyle(ResolverStyle.STRICT)
   val requestFormatter = DateTimeFormatter.ofPattern("uuuu-MM-dd").withResolverStyle(ResolverStyle.STRICT)
   val datePageFormatNoZero = DateTimeFormatter.ofPattern("d MMMM uuuu").withResolverStyle(ResolverStyle.STRICT)
 
-  def constructDate (day: Int, month: Int, year: Int): LocalDate = LocalDate.parse(s"$day/$month/$year", formatter)
+  def constructDate(day: Int, month: Int, year: Int): LocalDate = LocalDate.parse(s"$day/$month/$year", formatter)
```
Comment (jameshforster) [inline app/common/Dates.scala]: Should be using LocalDate not DateTime

File: test/assets/DateAsset.scala [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package assets
+
+import common.Dates
+import org.joda.time.DateTime
```
Comment (jameshforster) [inline test/assets/DateAsset.scala]: Same as above comments about LocalDate

**Comment by DavidWoodward37** [comment]
<h1> Merge with CGT-2036 on the back end </h1>

**Pull request merged**